src/main/java/com/github/cypher/sdk/User.java

```diff
@@ -202,7 +202,10 @@ public void removeLastActiveAgoListener(ChangeListener<? super Number> listener)
 		lastActiveAgo.removeListener(listener);
 	}
 
-
+	@Override
```

Comment:
If you override equals you should override hashCode too (otherwise HashMap etc wont function correctly).